### PR TITLE
Fixed Typos.

### PR DESCRIPTION
**Pull request by Tanvi05**

### CODE REVIEW

**Comment by prateekiiest** [comment]
Please describe the issue also.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+* Write clear and meaningful git commit messages (Do read https://chris.beams.io/posts/git-commit/)
+* Make sure your PR's description contains GitHub's special keyword references that automatically close the related issues when the PR is merged. (More info at https://github.com/blog/1506-closing-issues-via-pull-requests )
+* When you make very very minor changes to a PR of yours (like for example fixing a failing travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fixes. (Learn how to squash at https://davidwalsh.name/squash-commits-git )
+* When you're submitting a PR for a UI-related issue, it would be really awesome if you add the screenshot of that change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker.
 * If you are creating a issue, it would be awesome if you could add logs or crash reports as well as steps to reproduce it.
```
Comment (aashutoshrathi) [inline CONTRIBUTING.md]: Make it **an issue**

**Comment by aashutoshrathi** [comment]
ping @Tanvi05

**Comment by aashutoshrathi** [comment]
@jatinkatyal13 I think we should close this.

**Comment by jatinkatyal13** [comment]
Closing the PR !

**Pull request closed without merging**